db/version_set.cc

```diff
@@ -7137,6 +7137,20 @@ Status VersionSet::GetMetadataForFile(uint64_t number, int* filelevel,
 }
 
 void VersionSet::GetLiveFilesMetaData(std::vector<LiveFileMetaData>* metadata) {
+  if (!metadata) {
```

Comment:
I wish GetLiveFilesMetaData  was returning Status instead of void so that I could return Status::InvalidArgument if passed in vector ptr is null.